website/docs/r/emr_cluster.html.markdown

```diff
@@ -643,12 +646,12 @@ EOF
 
 resource "aws_iam_instance_profile" "emr_profile" {
   name  = "emr_profile"
-  roles = ["${aws_iam_role.iam_emr_profile_role.name}"]
+  roles = [aws_iam_role.iam_emr_profile_role.name]
```

Comment:
Looks like this has a minor merge conflict from the `aws_iam_instance_profile` resource `roles` argument removal 👍